src/PartnershipsConfig.ts

```diff
@@ -0,0 +1,25 @@
+const PartnershipsConfig = {
+    Country: [
```

Comment:
Better to make this a dictionary with key equal to the ISO3 code so we don't need to search through an array?

i.e.

```
Country: {
   "CAN": {
       "tableauKey": "CanadianTableau",
       ...
   }
}
```

Reply:
Nit: Could create a type for this config object, would serve as good documentation for what should be supplied when creating a new config